### PR TITLE
v1.1

### PR DESCRIPTION
**Pull request by laves**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,45 +1,130 @@
...
-    guard try VoiceProcessor.shared.hasPermissions() else {
-        print("Permissions denied.")
-        return
-    }
-
-    try VoiceProcessor.shared.start(
-        frameLength: 512, 
-        sampleRate: 16000, 
-        audioCallback: self.audioCallback)
+    try voiceProcessor.stop(frameLength: 512, sampleRate: 16000);
```
Comment (ErisMik) [inline README.md]:
```suggestion
    try voiceProcessor.stop();
```

File: .github/workflows/ios-tests.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        uses: actions/checkout@v3
+
+      - name: Install Cocoapods
+        run: gem install cocoapods
+
+      - name: Run Cocoapods
+        run: pod install
+
+      - name: XCode Build
+        run: xcodebuild test
```
Comment (ErisMik) [inline .github/workflows/ios-tests.yml:40]: Do we plan to run on AppCenter?

Comment (laves) [reply .github/workflows/ios-tests.yml:40]: I was thinking no for the voice processors. Our AppCenter is already bogged down enough with our products, I don't really want it to get any worse. Additionally, the unit tests, I believe, require a manual permission acceptance when run on a real device.

Comment (ErisMik) [reply .github/workflows/ios-tests.yml:40]: :+1:

**Review by ErisMik — APPROVED** [review]

**Review by ksyeo1010 — APPROVED** [review]

**Pull request merged**